tests/test_skycat.py

```diff
@@ -69,15 +76,17 @@ def test_get_wl_params(self):
         """
         for index in self.indexes:
             obj = self.skycat.objects[index]
-            galaxy_id = obj.get_native_attribute('galaxy_id')
-            row = self.df.query(f'galaxy_id == {galaxy_id}').iloc[0]
+            galaxy_id = obj.get_native_attribute("galaxy_id")
+            row = self.df.query(f"galaxy_id == {galaxy_id}").iloc[0]
             g1, g2, mu = obj.get_wl_params()
-            gamma1 = row['shear_1']
-            gamma2 = row['shear_2']
-            kappa = row['convergence']
-            self.assertAlmostEqual(g1, gamma1/(1. - kappa))
-            self.assertAlmostEqual(g2, gamma2/(1. - kappa))
-            self.assertAlmostEqual(mu, 1./((1. - kappa)**2 - (gamma1**2 + gamma2**2)))
+            gamma1 = row["shear_1"]
+            gamma2 = row["shear_2"]
+            kappa = row["convergence"]
+            self.assertAlmostEqual(g1, gamma1 / (1.0 - kappa))
+            self.assertAlmostEqual(g2, gamma2 / (1.0 - kappa))
+            self.assertAlmostEqual(
+                mu, 1.0 / ((1.0 - kappa) ** 2 - (gamma1**2 + gamma2**2))
+            )
```

Comment:
You ran black on this file I guess...?  There are a lot of content-free changes here.

Reply:
Yes, sorry. Forgot to turn my auto formatting off. I will remove the reformatting again

Reply:
Done